app/jobs/tracker_job.rb

```diff
@@ -0,0 +1,32 @@
+class TrackerJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(category, action, label, value, client_id, ga_id)
+    if ga_id.present?
```

Comment:
This check perhaps isn't necessary given you're checking `if AppSettings['settings.google_analytics_id'].present?` in your `tracker` method? No harm double checking though, I suppose.